tutorials/vbpr_text.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Copyright (c) Cornac Authors. All rights reserved.*\n",
+    "\n",
+    "*Licensed under the Apache 2.0 License.*\n",
+    "\n",
+    "# Visual Bayesian Personalizer Ranking (VBPR) with Text Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/PreferredAI/cornac/blob/master/tutorials/vbpr_text.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/PreferredAI/cornac/blob/master/tutorials/vbpr_text.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "In this tutorial, we will employ a recommendation algorithm designed to leverage item images to work with item text content.\n",
+    "The model of interest is [VBPR](https://arxiv.org/abs/1510.01784) that makes use of visual features extracted from item images using pre-trained CNN deep networks.\n",
+    "\n",
+    "What if our data doesn't come with item images but text, for example, we are working with [MovieLens dataset](https://grouplens.org/datasets/movielens/) that contains text plots of the movies. At the same time, we also want to work with VBPR model which is not originally designed for text data. To address that challenge, Cornac provides mechanisms to cross between modalities, in which here we focus on crossing from text to image to utilize image models for text-available datasets.\n",
```

Comment:
"What if our data doesn't come with item images but text`?`" For example, this is the case of the MovieLens dataset containing text movie plots.

Reply:
It's a phrase to state possibility, not a question.